ktor-client/ktor-client-core/common/src/io/ktor/client/features/HttpRedirect.kt

```diff
@@ -19,30 +20,39 @@ private val ALLOWED_FOR_REDIRECT: Set<HttpMethod> = setOf(HttpMethod.Get, HttpMe
  * [HttpClient] feature that handles http redirect
  */
 public class HttpRedirect {
+    private val _checkHttpMethod = atomic(true)
```

Comment:
Is it in our code style to prefix private members with an underscore?

Reply:
Yep